app/views/shared/provider/_user_widget.html.slim

```diff
@@ -68,3 +71,12 @@ javascript:
         window.hideAllToggleable()
       }
     })
+    .on('click', '.vert-nav-toggle', function (e) {
+      var shouldVertNavCollapse = !JSON.parse(store.isVerticalNavCollapsed)
```

Comment:
Do we need a `JSON.parse` here?

Reply:
To use a boolean yes, otherwise we could use `undefined / string` but it would be more volatile.

Reply:
Ah so it is stored as `undefined|'false'|'true'`?

But `JSON.parse('')`, `JSON.parse(undefined)` will raise an error

Reply:
Well you store some value so you expect it to be there and equal to what you store or not defined no?
Sadly we do not have type checker here
@didierofrivia wdyt?

Reply:
Well, type is going to be `string` regardless

Reply:
Wait it is merged ...

JSON.parse(undefined) is returning an error. How can this be OK? First time it is used it will be returning an error

Reply:
well, that's true. it should check if the stored value is not empty or undefined.

Reply:
I guess nobody verified it... Anyways this was hot-fixed in the main branch right after merging.